lib/fpm/command.rb

```diff
@@ -257,7 +257,7 @@ def initialize(*args)
   # Execute this command. See Clamp::Command#execute and Clamp's documentation
   def execute
     # Short-circuit if someone simply runs `fpm --version`
-    if ARGV == [ "--version" ]
+    unless (ARGV & ["-v", "--version"]).empty?
```

Comment:
This will print the version and exit if `-v` or `--version` appear anywhere in the flags. I don't think we want that. For example, this should not print the fpm version:


```
% bin/fpm -s empty -t rpm -n example -v 1.0
1.6.2
```

I have an idea on how to fix this, though.